app/lib/event_bus.rb

```diff
@@ -28,8 +28,9 @@ def self.reset!
 # wiring
 
 # automation
-EventBus.subscribe_job :message_thread_created, Automation::MessageThreadCreatedJob
-EventBus.subscribe_job :message_created, Automation::MessageCreatedJob
```

Comment:
toto som si teraz vsimol, potencialne sa moze stat problem ked v queue budu joby ktore nevie odrazu kod spracovat (lebo uz nema ako)

Reply:
jo, nasadíme tak, aby tam neboli.

Reply:
Zvykne sa to robit na 2 fazy, naprv nasadis nove, potom ked dobehnu joby, tak nasadis fix co vyhodi joby.

Reply:
ok, vrátil som